proxy/envoy/config.go

```diff
@@ -260,8 +261,24 @@ func buildHTTPListener(mesh *proxyconfig.ProxyMeshConfig, role proxy.Node, route
 		Config: FilterRouterConfig{},
 	})
 
+	service := ""
```

Comment:
qualify: mixerSourceService
Also, add a comment please, so that in future, when we axe this code, we know what to replace it with - and don't cause confusion again.

Reply:
Agree, added the comment - this is a revert, so the original name and code as we had it 2 weeks ago was used.

The only change I made was to make it compile.